<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentSetting</class>
 <widget class="QWidget" name="ParentSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton
{
	
	font: 75 18pt &quot;华康娃娃体W5(P)&quot;;
	
	color: rgb(133, 145, 255);
	background-color: rgba(145,111, 155, 23);
	border:none;
	border-radius:8px;
}
QPushButton:pressed
{
	
	background-color: rgba(145,111, 155,43);
	border:none;
	border-radius:8px;
}
QRadioButton
{
	color: rgb(133, 145, 255);
	font: 75 14pt &quot;华康娃娃体W5(P)&quot;;
}
QLabel
{
	color: rgb(133, 145, 255);
	font: 75 14pt &quot;华康娃娃体W5(P)&quot;;
}
QLineEdit
{
	color: rgb(133, 145, 255);
	font: 75 14pt &quot;华康娃娃体W5(P)&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">QFrame#frame_4
{
	border:1px solid black;
	border-radius:12px;
	background-color: rgb(255, 255, 255);
}
QPushButton
{
	
	font: 75 14pt &quot;华康娃娃体W5(P)&quot;;
	
	color: rgb(133, 145, 255);
	background-color: rgba(145,111, 155, 53);
	border:none;
	border-radius:8px;
}
QPushButton:pressed
{
	
	background-color: rgba(145,111, 155, 83);
	border:none;
	border-radius:8px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>55</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	
	font: 75 14pt &quot;华康娃娃体W5&quot;;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>54</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>471</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_5
{
border:1px solid black;
	border-radius:12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>95</width>
         <height>371</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="xUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>X+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="xDown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>X-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Y+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yDown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Y-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Z+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zDown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Z-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="small">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>430</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="huge">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>430</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>10.0</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="middle">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>430</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>141</width>
         <height>371</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="eOut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>挤出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eIn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>回抽</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="xHome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>XHome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yHome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>YHome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zHome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ZHome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Home">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>161</width>
         <height>301</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="printGcode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>打印Gcode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stoped">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recovery">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>恢复</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>预热</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="current_z">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>340</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>当前喷头高度</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>401</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_6
{
border:1px solid black;
	border-radius:12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>361</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QLineEdit" name="rectS">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>线径</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="rectA">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>回抽距离</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="flimant">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>回抽速度</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>打印高度</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="speeding">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="zvalues">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>温度(可选)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="tempture">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="extrudingSpeed">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Exculd</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>挤出宽度</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="widthE">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PrintSpeed</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
